rules/windows/network_connection/net_connection_win_susp_malware_callback_ports_uncommon.yml

```
@@ -0,0 +1,54 @@
+title: Communication To Uncommon Destination Ports
+id: 6d8c3d20-a5e1-494f-8412-4571d716cf5c
+related:
+    - id: 4b89abaa-99fe-4232-afdd-8f9aa4d20382
+      type: similar
+status: test
+description: Detects programs that connect to uncommon destination ports
+references:
+    - https://docs.google.com/spreadsheets/d/17pSTDNpa0sf6pHeRhusvWG6rThciE8CsXTSlDUAZDyo
+author: Florian Roth (Nextron Systems)
+date: 2017/03/19
+modified: 2023/12/11
+tags:
+    - attack.persistence
+    - attack.command_and_control
+    - attack.t1571
+logsource:
+    category: network_connection
+    product: windows
+detection:
+    selection:
+        Initiated: 'true'
+        DestinationPort:
+            - 8080
+            - 8888
+    filter_optional_sys_directories:
+        Image|contains:
+            - ':\Program Files\'
+            - ':\Program Files (x86)\'
+    filter_main_local_ips:
+        DestinationIp|startswith:
+            - '10.'
+            - '127.'
+            - '172.16.'
+            - '172.17.'
+            - '172.18.'
+            - '172.19.'
+            - '172.20.'
+            - '172.21.'
+            - '172.22.'
+            - '172.23.'
+            - '172.24.'
+            - '172.25.'
+            - '172.26.'
+            - '172.27.'
+            - '172.28.'
+            - '172.29.'
+            - '172.30.'
+            - '172.31.'
+            - '192.168.'
```

```suggestion
    filter_main_local_ips:
        DestinationIp|startswith:
            - '10.'
            - '127.'
            - '172.16.'
            - '172.17.'
            - '172.18.'
            - '172.19.'
            - '172.20.'
            - '172.21.'
            - '172.22.'
            - '172.23.'
            - '172.24.'
            - '172.25.'
            - '172.26.'
            - '172.27.'
            - '172.28.'
            - '172.29.'
            - '172.30.'
            - '172.31.'
            - '192.168.'    
    filter_optional_sys_directories:
        Image|contains:
            - ':\Program Files\'
            - ':\Program Files (x86)\'
```
filter_main* should come in front of filter_optional*